crates/core/src/db/datastore/locking_tx_datastore/mod.rs

```diff
@@ -1,1587 +1,23 @@
 mod btree_index;
 mod committed_state;
+mod datastore;
 mod mut_tx;
 mod sequence;
 mod state_view;
 mod table;
+mod tx;
 mod tx_state;
 
 pub use self::mut_tx::MutTxId;
+pub use datastore::{DataRef, Locking, RowId};
 pub use state_view::{Iter, IterByColEq, IterByColRange, StateView as _};
 
-use self::committed_state::CommittedState;
```

Comment:
It looks like the contents of this file were basically moved to `datastore.rs`. @Centril are there any material differences in `datastore.rs` or is it just a copy/paste?